tsconfig-dist.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "es2015",
```

Comment:
here it comes the pain. Webpack 2 is in beta and lots of people still use 1.x. ES2015 will at least require a properly configured babel -> ts-loader should be changed to awesome-ts-loader whatsoever; this is a really breaking change then. I think we should go for solution which works for all and set it to commonjs. Yes, tree-shaking won't e fully supported for now, but as long as there is only one `NgModule` with one `Directive` which is anyway imported when the module is imported I don't see any chance for tree shaking to anyhow improve the build. But we should keep the track on it to set it to be es2015 one day. Maybe with angular-sortablejs@2.0.0